src/generic_ui/scripts/ui.spec.ts

```diff
@@ -76,7 +76,6 @@ describe('UI.UserInterface', () => {
         userId: userId,
         name: userName,
         imageData: 'testImageData',
```

Comment:
Oh, yeah, unfortunately violates the JSON spec I think :(

Okay, I'm going to merge this in and call it good then!